src/components/events/EditSignup/DeleteSignup.tsx

```diff
@@ -38,13 +38,15 @@ const DeleteSignup = () => {
       })
     }
   }, [deleteSignup, event, navigate, paths, setSubmitting])
-
+  if (!event) {
+    return null
+  }
```

Comment:
Some kind of "Something bad happened"-component could be added here, but this currently will suffice IMO.